test/models/concerns/organization_extensions_test.rb

```diff
@@ -26,5 +26,21 @@ def test_audit_manifest_action
       end
       assert_equal @org.manifest_refreshed_at.to_i, current_time.to_i
     end
+
+    def test_clear_syspurpose_status
+      host = @org.hosts.first
+
+      HostStatus::Status.create(id: 1, host_id: host.id, type: 'Katello::PurposeStatus')
```

Comment:
`id` can be removed here - active record will create that for you!

with your new constant in the `manager` class you could also do this:

```
Katello::HostStatusManager::PURPOSE_STATUS.each do |status_class|
  HostStatus::Status.create(host: host, type: status_class.to_s)
end
```

Reply:
This refactor is totally up to you - just sharing a different approach. Should still remove `id` however!

Reply:
Did the refactor, looks cleaner now too. Thanks for the suggestion, learning new things with this PR :) 👍